### PR TITLE
SCSS.py: Add SCSS.py file

### PR DESCRIPTION
**Pull request by rishabhgarg25699**
Add a new python file in coalib/bearlib/languages/definitions/SCSS.py

Closes https://github.com/coala/coala/issues/5739

### CODE REVIEW

File: coalib/bearlib/languages/definitions/SCSS.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}
+encapsulators {
+	= {
+		'(': ')', '[': ']';
+	}
+}
+string_delimiter_escape {
+	= {
+		'"': '\\"', "'": "\\'";
+	}
```
Comment (shikharvaish28) [inline coalib/bearlib/languages/definitions/SCSS.py]: You need to change the formatting of the whole file. It's not as we expect it to be.

Comment (shikharvaish28) [reply coalib/bearlib/languages/definitions/SCSS.py]: This file is not in python at all

**Comment by shashwatmishra76** [comment]
@rishabhgarg25699 You are getting an assertion error under the method 
def test_get_all_languages_with_unknown(self) .It requires you to add 'SCSS 3.1, 3.2, 3.3, 3.4, 3.5, 4.0' to LANGUAGE_NAMES list.

File: .travis.yml [hunk 2]
```diff
@@ -12,15 +12,19 @@ stages:
   - name: unsupported
     if: branch = master AND type = push
 
+.disable_global: &disable_global
```
Comment (li-boxuan) [inline .travis.yml]: Remove those changes which are unrelated to your pr.

File: coalib/bearlib/languages/definitions/SCSS.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class SCSS:
+    extensions = '.scss'
+    comment_delimiters = '//'
```
Comment (li-boxuan) [inline coalib/bearlib/languages/definitions/SCSS.py]: should be a tuple

File: coalib/bearlib/languages/definitions/SCSS.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class SCSS:
+    extensions = '.scss'
```
Comment (li-boxuan) [inline coalib/bearlib/languages/definitions/SCSS.py]: should be a tuple

File: .travis.yml (hunk 2 above)
Comment (li-boxuan) [reply .travis.yml]: You didn't do what I said, and you are adding more irrelevant changes :(

I am pretty sure you are not using git command correctly

Fix the problem first. Next time before you push, use `gitk` or `gitg` tools, or use `git diff head~1` to check the changes within your commit.

File: coalib/bearlib/languages/definitions/SCSS.py (hunk 4 above)
Comment (li-boxuan) [reply coalib/bearlib/languages/definitions/SCSS.py]: This problem is still not fixed.

File: coalib/bearlib/languages/__init__.py [hunk 5]
```diff
@@ -7,6 +7,7 @@
 from .Language import Languages
 
 from .definitions.Unknown import Unknown
+from .definitions.Bash import Bash
```
Comment (shikharvaish28) [inline coalib/bearlib/languages/__init__.py]: Why are you adding ```Bash``` here? I don't think it's relevant here. IMO it's from  d5c0b79  which was earlier into this PR. Please check this.

File: tests/TestUtilities.py [hunk 6]
```diff
@@ -33,6 +33,7 @@
 
 # This list is sorted alphabetically
 LANGUAGE_NAMES = [
+    'Bash',
```
Comment (shikharvaish28) [inline tests/TestUtilities.py]: Same Here. ```Bash``` is not needed.

File: coalib/bearlib/languages/definitions/SCSS.py [hunk 7]
```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
```
Comment (shikharvaish28) [inline coalib/bearlib/languages/definitions/SCSS.py]: Blank space not needed.

File: tests/TestUtilities.py [hunk 8]
```diff
@@ -46,6 +47,7 @@
     'JavaScript Object Notation',
     'JavaServer Pages',
     'Jinja2',
+    'KornShell',
```
Comment (shikharvaish28) [inline tests/TestUtilities.py]: Not needed.

File: coalib/bearlib/languages/__init__.py (hunk 5 above)
Comment (rishabhgarg25699) [reply coalib/bearlib/languages/__init__.py]: It is done by me. When i sync my local repo with remote they are added automatically.
Even I did not know the meaning of bash then when i added this in this file.
And also now you open the coalib/bearlib/languages/__init__.py on github of coala, these all are already there. https://github.com/coala/coala/blob/master/coalib/bearlib/languages/__init__.py  SEE IN THIS LINK BASH IS ALREADY THERE.

Comment (rishabhgarg25699) [reply coalib/bearlib/languages/__init__.py]: I think that the github is comparing with old coala files.

File: coalib/bearlib/languages/definitions/SCSS.py (hunk 7 above)
Comment (rishabhgarg25699) [reply coalib/bearlib/languages/definitions/SCSS.py]: The gap is there in all the files. So i put this gap

File: tests/TestUtilities.py (hunk 8 above)
Comment (rishabhgarg25699) [reply tests/TestUtilities.py]: SEE THIS IS ALREADY PRESENT https://github.com/coala/coala/blob/master/tests/TestUtilities.py
It is comparing wrong  PLEASE SEE https://github.com/coala/coala/pull/5748/commits/01a27859071b51ffcd783eb3b283564b8f130bdb

File: coalib/bearlib/languages/definitions/SCSS.py (hunk 7 above)
Comment (shikharvaish28) [reply coalib/bearlib/languages/definitions/SCSS.py]: Since it's not needed, you will have to remove it.

File: tests/TestUtilities.py (hunk 8 above)
Comment (shikharvaish28) [reply tests/TestUtilities.py]: It's because you've used wrong ```git``` commands. 
It looks like that you've messed up your branch, I would suggest you to reset to the old commit and then make the changes once again.

File: coalib/bearlib/languages/definitions/SCSS.py [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+@Language
+class SCSS:
+    extensions = ('.scss')
```
Comment (li-boxuan) [inline coalib/bearlib/languages/definitions/SCSS.py]: This is wrong. `('.scss')` -> `'.scss',`

File: coalib/bearlib/languages/definitions/SCSS.py [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+    comment_delimiters = ('//')
```
Comment (li-boxuan) [inline coalib/bearlib/languages/definitions/SCSS.py]: Same as above

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5748/commits/d8eccb4e8026c75940f480cbea8f54b17d0307e6
